.github

```diff
@@ -0,0 +1,16 @@
+<!--  Thanks for sending a pull request!  Before submitting:
```

Comment:
this file should not be named `.github` I guess? ` .github/pull_request_template.md `

Reply:
yes, fixed